src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -55,8 +56,7 @@ class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLog
   private def simpleTransForms(input: Iterator[DatasetId], outputDir: File)
                               (printer: CSVPrinter): Try[FeedBackMessage] = input
     .map(simpleTransform(_, outputDir / UUID.randomUUID.toString, printer))
-    .collectFirst { case f @ Failure(_) => f }
-    .getOrElse(Success("OK"))
+    .failFastOr(Success("OK"))
 
   private def simpleTransform(datasetId: DatasetId, bagDir: File, printer: CSVPrinter): Try[FeedBackMessage] = {
```

Comment:
since you're not using the output anymore...

```suggestion
  private def simpleTransform(datasetId: DatasetId, bagDir: File, printer: CSVPrinter): Try[Unit] = {
```